From ae6a3f20345dac4b9daab8c39ac2d3fb3f2c21e3 Mon Sep 17 00:00:00 2001
From: Werner Koch <wk@gnupg.org>
Date: Fri, 21 Sep 2018 14:37:21 +0200
Subject: [PATCH Libgpg-error] syscfg: Add support for arc-unknown-linux-gnu

* src/mkheader.c (canon_host_triplet): Add to table.
--

Note that unknown in the above triplet is actually a wildcard for 4
part triplets.

Signed-off-by: Werner Koch <wk@gnupg.org>

Upstream-Status: Backport [http://git.gnupg.org/cgi-bin/gitweb.cgi?p=libgpg-error.git;a=commit;h=f4f0da74f526d7e35cedbc2e93454df6440dbfa5]
---
 src/mkheader.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/mkheader.c b/src/mkheader.c
index 7a38a1bec1a4..a4866e96ef43 100644
--- a/src/mkheader.c
+++ b/src/mkheader.c
@@ -92,7 +92,8 @@ canon_host_triplet (const char *triplet, int no_vendor_hack)
     {"i486-pc-linux-gnu", "i686-unknown-linux-gnu" },
     {"i586-pc-linux-gnu" },
     {"i686-pc-linux-gnu" },
-    {"arc-oe-linux-uclibc" }, /* Other CPU but same struct.  */
+    {"arc-oe-linux-gnu"    }, /* Other CPU but same struct.  */
+    {"arc-oe-linux-uclibc" }, /* and uclibc is also the same.  */
 
     {"i486-pc-gnu", "i686-unknown-gnu"},
     {"i586-pc-gnu"},
-- 
2.17.1

